frontend/components/ActivityDetails/InstallDetails/VppInstallDetailsModal/VppInstallDetailsModal.tsx

```
@@ -80,11 +80,11 @@ export const getStatusMessage = ({
   const formattedVerifyTimeout = secondsToDhms(vppVerifyTimeoutSeconds || 600);
   const displayTimestamp =
     ["failed_install", "installed"].includes(displayStatus || "") &&
-    commandUpdatedAt
+      commandUpdatedAt
```

ESLint

[error] Delete `··`

(prettier/prettier)